src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(hidden) {
+    super(hidden);
+    this.hidden = false;
+  }
+  hide(hidden) {
+    this.hidden = true;
+  }
```

Comment:
Herbivore has a method of hide, which changes the hidden property of the beast to the opposite value and helps to hide from carnivores

Reply:
You should change hidden property to the opposite value, not always to the true